cg/store/models.py

```diff
@@ -464,37 +464,38 @@ class Pool(Model):
 
 
 class Sample(Model, PriorityMixin):
+
+    application_version_id = Column(ForeignKey('application_version.id'), nullable=False)
+    beaconized_at = Column(types.Text)
+    capture_kit = Column(types.String(64))
+    comment = Column(types.Text)
+    created_at = Column(types.DateTime, default=dt.datetime.now)
+    customer_id = Column(ForeignKey('customer.id', ondelete='CASCADE'), nullable=False)
+    data_analysis = Column(types.String(16))
+    delivered_at = Column(types.DateTime)
+    deliveries = orm.relationship('Delivery', backref='sample')
+    downsampled_to = Column(types.BigInteger)
+    from_sample = Column(types.String(128))
```

Comment:
The alter table statement says this column is nullable

Reply:
must be nullable since we use same record for MIP and balsamic which lacks from_sample